src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -125,28 +182,8 @@ export function DAppsExplorerScreen() {
         )}
         {!loading && !error && categoriesById && (
           <AnimatedSectionList
-            ListHeaderComponent={
-              <>
-                <DescriptionView message={t('dappsScreen.message')} />
-                {featuredDapp && (
-                  <>
-                    <Text style={styles.sectionTitle}>{t('dappsScreen.featuredDapp')}</Text>
-                    <FeaturedDappCard dapp={featuredDapp} onPressDapp={onSelectDapp} />
-                  </>
-                )}
-
-                {dappFavoritesEnabled && (
-                  <>
-                    <Text style={styles.sectionTitle}>{t('dappsScreen.favoriteDapps')}</Text>
-                    <FavoriteDappsSection onPressDapp={onSelectDapp} />
-                  </>
-                )}
-
-                {(featuredDapp || dappFavoritesEnabled) && (
-                  <Text style={styles.sectionTitle}>{t('dappsScreen.allDapps')}</Text>
-                )}
-              </>
-            }
+            // @ts-ignore TODO: resolve type error
```

Comment:
spent way too long trying to get this error to go away and i couldn't do it, i think it might be that this reanimated lib type definition doesn't play nice with the RN types. it didn't inspire confidence to see this `ref` type named as `LegacyRef` in the reanimated type doc :/ open for ideas on how to resolve this / i think it might not be worth the effort to resolve this